react-native/index.d.ts

```diff
@@ -2563,8 +2563,18 @@ declare module "react" {
         ref?: Ref<DrawerLayoutAndroidStatic & ViewStatic>
     }
 
+    interface DrawerPosition {
```

Comment:
I feel it is different thing. the prop added here is a static while `drawerPosition` is a prop. but it will be nice to have drawerPosition updated too.